lib/jekyll-github-metadata/owner.rb

```diff
@@ -0,0 +1,86 @@
+# frozen_string_literal: true
+
+module Jekyll
+  module GitHubMetadata
+    class Owner
+      extend Forwardable
+
+      # Defines an instance method that delegates to a hash's key
+      #
+      # hash   - a symbol representing the instance method to delegate to. The
+      #          instance method should return a hash or respond to #[]
+      # key    - the key to call within the hash
+      # method - (optional) the instance method the key should be aliased to.
+      #          If not specified, defaults to the hash key
+      #
+      # Returns a symbol representing the instance method
+      def self.def_hash_delegators(hash, *methods)
```

Comment:
Since I introduced this, glad to move this to a module in a subsequent PR to DRY things up.

Reply:
Not a huge deal to me. This is a little different than the Repository class. 🤷‍♂️